modules/apps/object/object-rest-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Object REST API
 Bundle-SymbolicName: com.liferay.object.rest.api
-Bundle-Version: 19.10.2
+Bundle-Version: 20.0.0
```

Comment:
@victorKammerer about this:

1. put this change in the Semver commit
2. it leads to a break change, I believe we need to put a specific commit message here.